src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -22,6 +22,6 @@ public Country add(Country country) {
     @Override
     public Country get(Long id) {
         return countryDao.get(id).orElseThrow(() ->
-                new DataProcessingException("Can't get country by id " + id));
+                new RuntimeException("Can't get country by id " + id));
```

Comment:
This is OK to throw DataProcessingException

Reply:
fixed